internal/component/database_observability/mysql/component.go

```diff
@@ -63,6 +65,8 @@ type Arguments struct {
 var DefaultArguments = Arguments{
 	CollectInterval:               1 * time.Minute,
 	SetupConsumersCollectInterval: 1 * time.Hour,
+	LocksCollectInterval:          1 * time.Second,
+	LocksThreshold:                500 * time.Millisecond,
```

Comment:
highlighting these default values:
* Is 1 second too frequent? It seems like this would be the problematic lock duration threshold....
* locksThreshold is still set to 500ms to be a little bit more "aware" if there is a problem.

However both of these values may be too low. What do folks think?

Reply:
Done, I set it on the lower limit still. I put LocksCollectInterval at 30s and LocksThreshold at 1s. Let me know what you think.